argo-data-submission-wf/tests/input/file_pyega3.tsv

```diff
@@ -0,0 +1,2 @@
+type	name	format	size	md5sum		ega_file_id	ega_dataset_id	ega_run_id	ega_sample_id	ega_experiment_id
+file	EGAR00002324607_SLX-18928.UDP0002.bam	BAM	48308910340	597d50657b3d171953d518fb3e02bb7a		EGAF00004257597	EGAD00001007785	EGAR00002324607	EGAN00002634611	EGAX00002220414
```

Comment:
Just found the `path` field is missing from header.

Reply:
`path` isn't needed for pyega3 only the `EGAF`

Reply:
Yes. since `path` is defined as one of the `core` fields, should the field name still be present there to avoid errors?  
https://github.com/icgc-argo-workflows/data-processing-utility-tools/blob/main/payload-gen-seq-experiment/main.py#L72

Reply:
ah good point, I'll add back in the `file path`